contrib/debian/control.in

```diff
@@ -34,7 +34,7 @@ Depends: ${misc:Depends},
          shared-mime-info
 Recommends: python3,
 	    bolt,
-	    tpm2-tools,
+	    tpm2-tss,
```

Comment:
OK cool, I simply removed the tpm2-tools dependency for Debian and Fedora, hoping that it will work automagically (or be fixed by someone more knowledgeable in packaging for these distributions) ;)